samcli/commands/validate/validate.py

```diff
@@ -95,3 +114,40 @@ def _read_sam_file(template):
         sam_template = yaml_parse(sam_template.read())
 
     return sam_template
+
+def _lint(ctx, template):
```

Comment:
We are trying to add more typing into our repository, so can you please add typing for the parts that you have added?

Reply:
will look into it and add